readthedocs/projects/tasks.py

```diff
@@ -73,10 +73,7 @@
     after_vcs,
     before_build,
     before_vcs,
-    bulk_post_create,
-    bulk_post_delete,
     domain_verify,
-    files_changed,
```

Comment:
I believe this was actually used in our CDN app, so we might want to keep it around.

Reply:
Forgot about to check the rtd-ext, checking now

Reply:
In particular, we need this to be able to purge old files I believe. Perhaps we can just try another approach if we need CDN again though.